### PR TITLE
Header updates for the Roomify Travel Theme

### PR DESCRIPTION
**Pull request by salvoscala**
1. Move main menu to burger->full screen overlay (all the way to the left)
2. Remove logo and move property name to center of header
3. Give header a solid background and move image down

### CODE REVIEW

**Review by cecrs — COMMENTED** [review]
Love it.

File: themes/roomify/roomify_travel/assets/scripts/roomify_travel.js [hunk 1]
```diff
@@ -50,6 +49,21 @@ Drupal.behaviors.roomifyTravelScripts = {
         $(this).closest('.slide__caption').hide();
       }
     });
+
+    $('#roomify-main-menu-overlay .closebtn').once().click(function(e) {
+      //e.preventDefault();
```
Comment (acrollet) [inline themes/roomify/roomify_travel/assets/scripts/roomify_travel.js]: @salvoscala let's get rid of these commented lines? also a quick comment for what the code is doing would be nice :)

File: themes/roomify/roomify_travel/color/preview.php [hunk 2]
```diff
@@ -1,7 +1,12 @@
 <div id="preview">
   <!-- Header. -->  
   <header id="header">
-
+    <span class="main-menu">
+      <a class="roomify-main-menu-hamburger" href="#menu-toggle">
```
Comment (acrollet) [inline themes/roomify/roomify_travel/color/preview.php:5]: mmmmmm... hammmmmmmburgers....

**Review by acrollet — CHANGES_REQUESTED** [review]
CR passes after the minor cleanup

**Review by acrollet — APPROVED** [review]
CR passes

**Pull request merged**